src/object/srv_ec_aggregate.c

```diff
@@ -2715,8 +2720,32 @@ cont_ec_aggregate_cb(struct ds_cont_child *cont, daos_epoch_range_t *epr,
 	agg_reset_entry(&ec_agg_param->ap_agg_entry, NULL, NULL);
 
 retry:
+	epoch.oe_value = epr->epr_hi;
+	rc = dtx_begin(cont->sc_hdl, &dti, &epoch, 0, cont->sc_pool->spc_map_version, &oid,
```

Comment:
the dth is just for returning tbd_list or for other reasons?

Reply:
For this case, the `dth` is just for returning `tbd_list`. But for other cases, it will have more usage.